src/libraries/Common/src/Interop/Windows/Version/Interop.VerQueryValue.cs

```diff
@@ -10,6 +10,6 @@ internal static partial class Version
     {
         [LibraryImport(Libraries.Version, EntryPoint = "VerQueryValueW", StringMarshalling = StringMarshalling.Utf16)]
         [return: MarshalAs(UnmanagedType.Bool)]
-        internal static partial bool VerQueryValue(IntPtr pBlock, string lpSubBlock, out IntPtr lplpBuffer, out uint puLen);
+        internal static unsafe partial bool VerQueryValue(void* pBlock, string lpSubBlock, out IntPtr lplpBuffer, out uint puLen);
```

Comment:
```suggestion
        internal static unsafe partial bool VerQueryValue(void* pBlock, string lpSubBlock, out void* lplpBuffer, out uint puLen);
```
+ matching changes at callsites

Reply:
I'd actually done this initially but undid it as some of the use sites then turned around and cast it to an IntPtr.  But since you brought it up, I redid it.